magenta/interfaces/midi/midi_hub_test.py

```diff
@@ -345,7 +345,7 @@ def testStartCapture_IsDrum(self):
     captor = self.midi_hub.start_capture(120, start_time)
 
     # Channels are 0-indexed in mido.
```

Comment:
is this comment still true?

Reply:
Yes, I just thought the one-indexed drum channel was 9 when it is actually 10.